### PR TITLE
fix: rounded corner login page

### PR DESCRIPTION
**Pull request by imchell**
Before:
login before

After:
login after

### CODE REVIEW

**Comment by hsluoyz** [comment]
@leo220yuyaodog plz review

File: web/src/App.less [hunk 1]
```diff
@@ -134,6 +126,11 @@
   box-sizing: border-box;
   margin: 0 auto;
   position: relative;
+  box-shadow: 0 0 20px rgb(0 0 0 / 20%);
```
Comment (leo220yuyaodog) [inline web/src/App.less:129]: Please don't modify class `.login-panel`. It't will cause many other problem that incompatible with other feature about the custom login page ui. For beatifying the default login page.  As far as possible just modify the class `.login-form`

image

Comment (imchell) [reply web/src/App.less:129]: Looking into it.

Comment (imchell) [reply web/src/App.less:129]: Hi @leo220yuyaodog I've double checked but it seems I didn't modify `.login-panel`? How to reproduce your result shown in the screenshot?

Comment (leo220yuyaodog) [reply web/src/App.less:129]: > Hi @leo220yuyaodog I've double checked but it seems I didn't modify `.login-panel`. How to reproduce your result shown in the screenshot? What I have changed is `.login-content` only.

Custom the application UI follow the document https://casdoor.org/docs/application/ui-customization

Comment (imchell) [reply web/src/App.less:129]: > Custom the application UI follow the document https://casdoor.org/docs/application/ui-customization

I see the problem here. Does the style filled in Form CSS in casdoor overwrite the default style in `.login-panel`? If that is the case, can we just assign `.login-panel` a more unified default style?

Comment (hsluoyz) [reply web/src/App.less:129]: @imchell , @leo220yuyaodog wrote this part of code and is more familar with it. I think it's more suitable for you to create an issue instead and let him fix it.

**Pull request closed without merging**